### DISCUSSION TITLE
Is it currently possible to update TableSortLabel Icon component

### DISCUSSION THREAD

**Discussion by syedqasimahmed** [KevinVandy/material-react-table#130] [Q&A] [answered]
I am currently using this library in my project. I want to change Sort Icon besides column names. 

Is this currently possible If not 
Should I make a PR for that

**Comment by KevinVandy** [comment]
It's currently not possible, because I didn't think anyone would want to do that. Curios, what would you change it to?

**Reply by syedqasimahmed** [reply]
I want to change it to custom icon Component. So It is more customizeable.

**Reply by KevinVandy** [reply]
This has been added in `v1.2.9`

See icon docs to learn how https://www.material-react-table.com/docs/guides/customize-icons

**Comment by KevinVandy** [comment, marked as answer]
Yes, you can customize the `ArrowDownwardIcon` in the `icons` prop

See here https://www.material-react-table.com/docs/guides/customize-icons

**Reply by syedqasimahmed** [reply]
Thanks
